### ISSUE TITLE
default_cpu_entry option does not work

### ISSUE THREAD

**Issue by tkrause** [ClementTsang/bottom#1539]
### Checklist

- [X] I've looked through the [troubleshooting docs](https://clementtsang.github.io/bottom/nightly/troubleshooting), [the known problems list](https://clementtsang.github.io/bottom/nightly/support/official/#known-problems), and  [existing open issues](https://github.com/ClementTsang/bottom/issues?q=is%3Aopen+is%3Aissue) for similar issues.

### What operating system and version are you using?

MacOSX 14.5

### What architecture are you using?

arm64

### What terminal(s) are you running bottom on that are experiencing the problem?

iterm2

### What filesystem(s) are you using?

APFS

### What version of bottom are you running?

0.10.1

### How did you install bottom?

brew install bottom

### Describe the issue

When using the config file OR flag: `--default_cpu_entry=avg`, nothing happens the CUI stays on `All` for the CPU graph.
Doing a quick search on Github, it looks like the parameter is defined and validation occurs. However, it doesn't appear to be referenced assuming this isn't fully implemented.

### What is the expected behaviour?

The CPU graph should switch to avg displaying

### What is the actual behaviour?

<img width="420" alt="image" src="https://github.com/user-attachments/assets/86da341e-5d74-4566-9d63-c43651e14f07">

### How can we reproduce this?

btm --default_cpu_entry="avg"

**Comment by nickchomey** [comment]
Not sure if I should create a new issue or not, but this seems to be happening again.

**Comment by ClementTsang** [comment]
Hm, will take a look. Thanks for reporting.

**Comment by ClementTsang** [comment]
I'll reopen just to track it I guess 😅

**Comment by nickchomey** [comment]
wow, thats the quickest response time I've ever seen!

If it helps at all, 
version 0.11.3
ubuntu 24.04
amd64 

bottom_x86_64-unknown-linux-musl.tar.gz (installed via [mise](https://mise.jdx.dev), via [aqua](https://github.com/aquaproj/aqua))

**Comment by ClementTsang** [comment]
Hm, it seems to be working fine if I try using it in `main`?

```bash
cr -- --default-cpu-entry=all
```

<img width="146" height="348" alt="Image" src="https://github.com/user-attachments/assets/bfe96373-369b-4e49-af51-383e76a3286d" />

```bash
cr -- --default-cpu-entry=avg
```

<img width="235" height="358" alt="Image" src="https://github.com/user-attachments/assets/c8999afb-3fb8-4c37-8556-108015506b31" />

**Comment by ClementTsang** [comment]
Oh wait, let me also test the config...

**Comment by ClementTsang** [comment]
Yeah, works fine for config too. Going to mark this as closed then as it seems to be fine.
